### DISCUSSION TITLE
hyperparameter funing tools for flax

### DISCUSSION THREAD

**Discussion by wztdream** [google/flax#1817] [General] [answered]
Hi,
Searching hyperparameters is time and resource consuming, It will be better to have tools  like [ray](https://docs.ray.io/en/latest/tune/tutorials/overview.html) to assist this process, but ray seems not support flax now.  So is there available hyper parameter tuning tools for flax now?

**Comment by mattiasmar** [comment, marked as answer]
SigOpt could do it for you as a service. 

Their service would generate the set of hyperparameters, your code would evaluate the set, and report back results to SigOpt, and you'd get a new set of parameters in return according to the Bayesian Optimization that SigOpt runs.

**Reply by wztdream** [reply]
That seems great, I will have a try thank you
